classes/hovercards/class-hovercards.php

```diff
@@ -2,16 +2,13 @@
 
 namespace Automattic\Gravatar\GravatarEnhanced\Hovercards;
 
-class Hovercards {
-	/**
-	 * @var string
-	 */
-	const OPTION_HOVERCARDS = 'gravatar_hovercards';
+use Automattic\Gravatar\GravatarEnhanced\Module;
 
+class Hovercards implements Module {
 	/**
 	 * @var string
 	 */
-	const FILTER_GRAVATAR_HOVERCARDS_MODULE_ENABLED = 'gravatar_enhanced_hovercards_module_enabled';
```

Comment:
Do we need to provide any kind of backwards compatibility support for this removed filter?

Reply:
Good question. This was removed by mistake. I was going to add a generic `gravatar_enhanced_*_module_enabled` filter and check on every module. But I ended up going with the whitelist approach. Reverting.